### ISSUE TITLE
DPL-nnn - Story 1 - repurpose existing ONT code

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-ui#705]
**User story**
The current ONT pipeline in the ui is related to the experimental Covid work and is no longer relevant. We need to remove everything that is not relevant and leave a skeleton to add the ONT pipeline

**Who are the primary contacts for this story**
Steve / James

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
sanger/traction-service#707
